internal/app/backend/backend_service.go

```diff
@@ -171,7 +170,7 @@ func synchronizeRecv(ctx context.Context, syncStream synchronizerStream, m *sync
 			}
 			stats.Record(ctx, totalBytesPerMatch.M(int64(proto.Size(match))))
 			stats.Record(ctx, ticketsPerMatch.M(int64(len(match.GetTickets()))))
-			err = stream.Send(&pb.FetchMatchesResponse{Match: match})
+			err := stream.Send(&pb.FetchMatchesResponse{Match: match})
```

Comment:
why did you decide to create a new err variable here?

Reply:
that was an idea i had about moving all that if block out but wasn't completely sure about, and that colon just escaped me